Package.swift

```diff
@@ -8,7 +8,7 @@ let package = Package(
     ],
     dependencies: [
         // 💧 A server-side Swift web framework.
-        .package(url: "https://github.com/vapor/vapor.git", from: "4.0.0-rc"){{#fluent}},
+        .package(url: "https://github.com/vapor/vapor.git", from: "4.5.1"){{#fluent}},
```

Comment:
Putting such a specific version here seems kind of weird to me. Theoretically everything in this template _is_ compatible with version 4.0.0, so why wouldn't we put that? It seems like if we put something more specific we should be sure it is updated regularly. 

I may be way overthinking this though >_<. What does everyone else think?

Reply:
Would it be overkill (probably a yes) to make it a template variable and just pick up the latest released version whenever someone initialises a new repo? Would only be a simple GH API Req

Reply:
Seems a bit excessive given that it's what `from` and `upToNextMajor` and `upToNextMinor` are there for in SPM to begin with 🤷‍♀️

Reply:
Agreed. In that case I'd just put down `4.0.0` :)

Reply:
@gwynne updated!